src/CCDReduction.jl

```diff
@@ -11,7 +11,8 @@ export subtract_bias,
        trim,
        trimview,
        crop,
-       cropview
+       cropview,
+       fits_indices
```

Comment:
I wouldn't export this

Reply:
Right, I don't think it is required for the end user!